packages/manager/apps/pci-ai-notebooks/src/pages/notebooks/auth/Auth.page.tsx

```diff
@@ -0,0 +1,10 @@
+const Auth = () => {
+  return (
+    <div>
+      <h2>Hello Auth Page</h2>
+      <p>This is the future auth page</p>
+    </div>
+  );
+};
+
+export default Auth;
```

Comment:
I suppose this will be added in an upcoming PR

Reply:
Yes, will be done in another branch